rukundoeric/calculator
# Tests

## What does this do? 
*  Tests
## Description of Task to be completed? 
*  In this part of the project, I wrote tests for everything I implemented on the calculator project.
## How should this be manually tested? 
NOTE: Make sure you have Node.js and npm or yarn installed on your PC.
- Clone this repo: `git clone https://github.com/rukundoeric/calculator.git`
- Navigate into the project directory: `cd calculator`
- Install all packages: `yarn install or npm install`
- Run the tests: `yarn test or npm test`
